ghost/members-events-service/lib/EventStorage.js

```
@@ -40,6 +40,7 @@ class EventStorage {
         });
 
         domainEvents.subscribe(SubscriptionCreatedEvent, async (event) => {
+            console.log('SubscriptionCreatedEvent > subscriptionId', event.data.subscriptionId);
```

ESLint

[error] Unexpected console statement.

(no-console)